collaborations/blood-hunter-2022/bloodhunter-profanesoul.xml

```diff
@@ -0,0 +1,654 @@
+			<p class="indent"><strong><em>The Undead.</em></strong> You can cast <i>speak with dead</i>.</p>
+			<p class="indent"><strong><em>The Undying.</em></strong> You can cast <i>bestow curse</i>.</p>
+		</description>
+		<sheet display="false">
+			<description></description>
+		</sheet>
+		<rules>
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_ARCHFEY" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_ARCHFEY" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_CELESTIAL" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_CELESTIAL" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_FATHOMLESS" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_FATHOMLESS" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_FIEND" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_FIEND" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_GENIE" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_GENIE" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_GREAT_OLD_ONE" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_GREAT_OLD_ONE" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_HEXBLADE" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_HEXBLADE" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_UNDEAD" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_UNDEAD" />
+			<grant type="Archetype Feature" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_UNSEALED_ARCANA_UNDYING" requirements="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_OTHERWORLDLY_PATRON_UNDYING" />
+		</rules>
+	</element>
+	
+	<element name="Blood Curse of the Souleater" type="Archetype Feature" source="Blood Hunter (2022)" id="ID_DDB_BH22_ARCHETYPE_FEATURE_PROFANE_SOUL_BLOOD_CURSE_OF_THE_SOULEATER">
+		<description>
+			<p>Starting at 18th level, you learn to siphon the life energy from your fallen prey. You gain the Blood Curse of the Soul Eater for your Blood Maledict feature. This doesn’t count against your number of blood curses known.</p>
+		</description>
+		<sheet display="false">
+			<description>You gain the Blood Curse of the Souleater for your Blood Maledict feature. This does not count against your number of blood curses known.</description>
+		</sheet>
+		<rules>
+			<grant type="Class Feature" id="ID_DDB_BH22_CLASS_FEATURE_BLOOD_HUNTER_BLOOD_CURSE_OF_THE_SOULEATER" />
+		</rules>
+	</element>
+
+	<!--Otherworldly Patrons-->
```

Comment:
As much as I like the sheet descriptions, it will still take up space. So either make them more explicit with who their patron is or remove it entirely (display="false").